converters/google/convert.go

```diff
@@ -179,7 +180,30 @@ func (c *Converter) Schemas() map[string]*schema.Resource {
 }
 
 // AddResource converts a terraform resource and stores the converted asset.
-func (c *Converter) AddResource(r TerraformResource) error {
+func (c *Converter) AddResource(rc *tfplan.ResourceChange) error {
+	// Compatibility shim: silently do nothing with deleted resources.
+	if len(rc.Change.Actions) == 1 && rc.Change.Actions[0] == "delete" {
```

Comment:
should we add an `isDelete` helper similar to `isCreate`?

Reply:
yes, but I think that will fit better in one of the future PRs.

Reply:
After thinking about this a bit more I think I need to flip this around, since this gate would let through things that we don't necessarily want to handle (specifically `["no-op"]`, `["read"]`, and `["create", "delete"]`).

Reply:
Changed this in b3f75001. I left `IsDelete` for a future PR since we won't be using it yet.